cmake/gtest/CMakeLists.txt.in

```diff
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 2.8.4)
+
+project(googletest-download NONE)
+
+include(ExternalProject)
+ExternalProject_Add(googletest
+	URL https://github.com/google/googletest/archive/master.zip
```

Comment:
This should be versioned.

Reply:
What about `GIT_REPOSITORY` and `GIT_TAG` instead of `URL`? Newer cmake can do this with a shallow clone.

Reply:
@MaEtUgR , in case it is helpful, latest is 1.8.1:
```
URL https://github.com/google/googletest/archive/release-1.8.1.tar.gz
URL_HASH SHA1=152b849610d91a9dfa1401293f43230c2e0c33f8
```

Reply:
I totally agree, 
- Cloning the repo works as well but I had the impression it wasn't shallow because it took so much longer compared to just using the zip, would have to check in more detail.
- I didn't use a release for my testing because my pr https://github.com/google/googletest/pull/2101 was just merged and I wanted it in for my Cygwin testing. I'll put a release or at least fixed commit before removing the draft 👍

Reply:
I found out it's possible to download a zip from a specific version without tag from GitHub. I'll add this now.

Reply:
Indeed ^ , just add this instead of the line you had:
```
URL https://github.com/google/googletest/archive/release-1.8.1.tar.gz
URL_HASH SHA1=152b849610d91a9dfa1401293f43230c2e0c33f8
```

Reply:
I chose `URL https://github.com/google/googletest/archive/8b6d3f9c4a774bef3081195d422993323b6bb2e0.zip`. It's also a version from the master branch just without a tag.